tardis/plugins/prometheusmonitoring.py

```diff
@@ -0,0 +1,77 @@
+from ..configuration.configuration import Configuration
+from ..interfaces.plugin import Plugin
+from ..interfaces.state import State
+from ..interfaces.siteadapter import ResourceStatus
+from ..utilities.attributedict import AttributeDict
+
+import logging
+from aioprometheus import Service, Gauge
+
+
+class PrometheusMonitoring(Plugin):
+    """
+    The :py:class:`~tardis.plugins.prometheusmonitoring.PrometheusMonitoring`
```

Comment:
You can also use a relative path for sphinx links.

```suggestion
    The :py:class:`~.PrometheusMonitoring`
```